src/utils/browser.js

```diff
@@ -21,5 +21,17 @@ export default {
             let m = num.toExponential().match(/\d(?:\.(\d*))?e([+-]\d+)/)
             return num.toFixed(Math.max(0, (m[1] || '').length - m[2]))
         }
+    },
+    getLength(str) {
```

Comment:
we'd better create a new file named "common.js" and put all common function in this file.